thundra/plugins/invocation/invocation_plugin.py

```diff
@@ -78,6 +78,26 @@ def get_response_status(self, plugin_context):
             return None
         return status_code
 
+    def inject_step_function_info(self, plugin_context, outgoing_trace_links):
+        try:
+            response = plugin_context['response']
+            event = plugin_context['request']
+            if config.is_step_function():
+                trace_link = str(uuid.uuid4())
+                plugin_context['response_trace_link'] = trace_link
```

Comment:
Seems that no need to put generated `trace_link` into `plugin_context`. Because as far as I see, `plugin_context['response_trace_link']` is only accessed inside `inject_step_function_info` here. So I think we can remove 
```python
plugin_context['response_trace_link'] = trace_link
``` 
and change
```python
outgoing_trace_links["outgoingTraceLinks"].append(plugin_context['response_trace_link'])
```
as 
```python
outgoing_trace_links["outgoingTraceLinks"].append(trace_link)
```